### DISCUSSION TITLE
Trailing pro edition

### DISCUSSION THREAD

**Discussion by Robw94** [cyrilletuzi/vscode-angular-schematics#934] [Q&A] [answered]
Hi, 

I am trailing the pro edition, Just created a brand new angular 18 app to play about with it and noticed the 'Reactive Service' (Signal) is now missing? I only have the RxJs option

**Comment by cyrilletuzi** [comment]
Hello @Robw94,

The signal version of the reactive service schematic is enabled when the extension can detect that `@angular/core` version >= 16 is present in `pakckage.json`. Is that the case in this project, or are you using a special package manager?

**Comment by Robw94** [comment]
I created a new app using the command ng new angular-playground, no special packages.

`{
  "name": "angular-playground",
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "watch": "ng build --watch --configuration development",
    "test": "ng test"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "^18.2.0",
    "@angular/common": "^18.2.0",
    "@angular/compiler": "^18.2.0",
    "@angular/core": "^18.2.0",
    "@angular/forms": "^18.2.0",
    "@angular/platform-browser": "^18.2.0",
    "@angular/platform-browser-dynamic": "^18.2.0",
    "@angular/router": "^18.2.0",
    "rxjs": "~7.8.0",
    "tslib": "^2.3.0",
    "zone.js": "~0.14.10"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "^18.2.8",
    "@angular/cli": "^18.2.8",
    "@angular/compiler-cli": "^18.2.0",
    "@types/jasmine": "~5.1.0",
    "jasmine-core": "~5.2.0",
    "karma": "~6.4.0",
    "karma-chrome-launcher": "~3.2.0",
    "karma-coverage": "~2.2.0",
    "karma-jasmine": "~5.1.0",
    "karma-jasmine-html-reporter": "~2.1.0",
    "typescript": "~5.5.2"
  }
}
`

**Comment by cyrilletuzi** [comment, marked as answer]
@Robw94 I sent you an email to the address you indicated in your Gumroad subscription, it will be easier and faster to help you.
